### PR TITLE
Implement environment variable for bot's token

### PR DESCRIPTION
**Pull request by mobil6000**
I implemented the ability to setup a bot's token using an environment variable. 
the corresponding instructions have been added to the readme file.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
+python-dotenv
```
Comment (alex19EP) [inline requirements.txt]: should we use file? i think environment variable is enough.
@evgeniy-tulyakov what are your arguments?

**Review by alex19EP — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (mobil6000) [reply requirements.txt]: creating a ".env" file  is not a requirement. 
The bot loads this file only if it is created. 
creating this file absolvs us from permanent use of the "export" command.

**Review by alex19EP — APPROVED** [review]
LGTM. thanks for contribution.

**Comment by alcortazzo** [comment]
Thank you for contributing

**Pull request merged**